SharpPcap/LibPcap/CaptureFileWriterDevice.cs

```diff
@@ -111,11 +111,6 @@ public override void Close()
         /// </summary>
         public override void Open(DeviceConfiguration configuration)
         {
-            if (configuration.Snaplen > Pcap.MAX_PACKET_SIZE)
-            {
-                configuration.RaiseConfigurationFailed("snaplen", new InvalidOperationException("Snaplen > Pcap.MAX_PACKET_SIZE"));
-            }
-
```

Comment:
@chmorgan after checking libpcap docs, there is no statement that says spanlen must not be larger than 65536.
All what the docs says, if you set it to 65536, then it is big enough for any known traffic.

Reply:
I think that is the case. I'm not sure it always was or not. Might have been from the original library even.